.rubocop_todo.yml

```diff
@@ -1,11 +1,15 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-02-13 12:54:33 -0500 using RuboCop version 0.79.0.
+# on 2020-03-02 18:30:42 -0800 using RuboCop version 0.74.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 4
+Metrics/AbcSize:
+  Max: 37
```

Comment:
this is weird - it's because of something to do with `Module.included` which is an inherited method.  Rubocop didn't complain about the same thing when it was defined as part of the Class.